test/integration/master/graceful_shutdown_test.go

```diff
@@ -92,7 +92,8 @@ func TestGracefulShutdown(t *testing.T) {
 	}
 	body.Close()
 	respErr := <-respErrCh
-	if err != nil {
+	t.Logf("respErr %v", respErr)
+	if respErr.err != nil {
 		t.Fatal(err)
```

Comment:
This should also be `t.Fatal(respErr.err)`, but I'm not sure why it's returning an error or what that error is

Reply:
> This should also be `t.Fatal(respErr.err)`, but I'm not sure why it's returning an error or what that error is

It doesn't seem to affect the result.

Reply:
It's quite confusing why we need to change the test...

Reply:
These are mistakes in the test that we are only noticing now because the test is failing, so we need to fix it to check the proper error and then report the proper error on a failure. Right now, the test is failing, but the `fatal` call is only reporting `<nil>` as the error, so fixing the check and reporting is important